### PR TITLE
Update resources

### PR DESCRIPTION
**Pull request by purplem1lk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,11 @@ Short description of the resource. It’s recommended to copy and paste the best
 - Topics: Topic 1, Topic 2, Topic 3
 ```
 
+---
+
```
Comment (thetaPC) [inline README.md:39]: This needs to be removed.

File: README.md [hunk 2]
```diff
@@ -57,7 +59,81 @@ You will learn how to get ahead, stand out, become more visible, and advocate fo
...
+#### [Digital Life Design](http://dld-conference.com/)
+
+DLD (Digital-Life-Design) have created an interdisciplinary and internationally connected platform for people eager to change the world in the digital era. They host global events (DLD All Stars, DLD Sync, and many more)that are geared towards young entrepreneurs looking to get an all-around look at coding, data, business, social media, and more.
+
+- Topics: Entrepreneutrship, Events
+
+#### [Lesbians Who Tech & Allies](https://lesbianswhotech.org/pridesummit2021/)
+
+Lesbians Who Tech & Allies is a community of LGBTQ women, non-binary and trans individuals in and around tech with goals of inclusivity and diversity in tech. They offer a virtual queer inclusive summits such as (Not IRL) Pride Summit with speakers from all backgrounds and sponsor inclusivity in all of their talks. They also offer coding scholarships through their bootcamp partners and a jobs board through https://we.include.io/.
```
Comment (thetaPC) [inline README.md]: https://we.include.io/ convert this to a link with an appropriate text.

**Review by thetaPC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (purplem1lk) [reply README.md:39]: @thetaPC I added this on purpose to separate the Contributing section from when the Resources start. Is it confusing to have the line here?

Comment (thetaPC) [reply README.md:39]: You have a great point and I like the reasoning. Let's keep it in.

**Review by thetaPC — APPROVED** [review]
LGTM

**Pull request merged**